shell/browser/api/electron_api_browser_window.cc

```diff
@@ -77,6 +77,7 @@ BrowserWindow::BrowserWindow(gin::Arguments* args,
       WebContentsView::Create(isolate, web_preferences);
   DCHECK(web_contents_view.get());
   window_->AddDraggableRegionProvider(web_contents_view.get());
+  web_contents_view_.Reset(isolate, web_contents_view.ToV8());
```

Comment:
This means that the WebContents can't be disposed of without destroying its originating BrowserWindow, right?

Reply:
This is already true, I believe.